src/App.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data.js';
+import CommentSection from './components/CommentSection/CommentSection.js';
+import PostContainer from './components/PostContainer/PostContainer.js';
+import {SearchBar} from './components/SearchBar/SearchBar.js';
+
+class App extends Component {
+ 
+	constructor(){
+		super();
+		this.state = {
+			dummyData: dummyData,
+			inputText:""
+		};
+	}
+
+
+ render() {
+    return (
+      <div className="App">
+			<SearchBar/>
+			<CommentSection/>
+			{dummyData.map( (post, index) => (<PostContainer userPost={post} key={post.username + index} />))}
+			<div>{console.log(dummyData)}</div>
```

Comment:
If you want to `console.log` something in the `render` method, you don't have to do it that way, you can just do it outside of the `return` statement and you don't need to wrap it in a `div`